tests/karate/src/test/java/be/createLAO/create.feature

```diff
@@ -2,14 +2,39 @@
 Feature: Create a pop LAO
 
   Background:
-        ## This is feature will be called  to
+        # This is feature will be called  to test an LAO creation
         # This call makes this feature and server.feature share the same scope
         # Meaning they share def variables, configurations ...
         # Especially JS functions defined in server.feature can be directly used here thanks to Karate shared scopes
-        # * call wait <timeout>
-        # * karate.set(varName, newValue)
     * call read('classpath:be/utils/server.feature')
 
+  Scenario: Create Lao request with empty lao name should fail with an error response
+    Given string  emptyNameReq = read('classpath:data/lao/bad_lao_create_empty_name.json')
```

Comment:
I think it would be good to use only one implementation of a web socket throughout the whole codebase.

It might create confusion in the future.

This doesn't need to be done in this PR in my opinion, but it should be noted.
What do you think ?

Reply:
Yes, this is a great consistency idea, I will address this in my next PR.